content/models.py

```diff
@@ -27,3 +27,17 @@ class OurStoryTitle(models.Model):
 
     def __str__(self):
         return self.title
+
+class ChildrensProgram(models.Model):
+	pagetitle = models.CharField(blank=True, null=True, max_length=100)
```

Comment:
Just remove `blank=True` and `null=True` in one of the titles, if I'm understanding you correctly